module/VuFind/src/VuFind/View/Helper/Root/Icon.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Make link view helper
```

Comment:
Copied and pasted comment.

Reply:
You marked this one as resolved, but it still says "make link" instead of "icon." Did something not get pushed?